README.md

```diff
@@ -73,3 +73,6 @@ Participation in the Kubernetes community is governed by the [Kubernetes Code of
 
 [owners]: https://git.k8s.io/community/contributors/guide/owners.md
 [Creative Commons 4.0]: https://git.k8s.io/website/LICENSE
+
+
+TODO add some lines about what to find ./hack
```

Comment:
a `tree` command output and the description of each file could be helpful on this PR

Reply:
I think the need to rework of the readme is bigger than this PR, so I created a new [issue](https://github.com/kubernetes-sigs/kpng/issues/124). (This is completely not me being too lazy for writing docs. :) )